src/runtime/write_debug_image.cpp

```diff
@@ -101,14 +101,65 @@ WEAK bool has_tiff_extension(const char *filename) {
     return *f == '\0';
 }
 
+int32_t get_data_index(int32_t dim0, int32_t dim1, int32_t dim2, int32_t dim3,
```

Comment:
Also, this function needs to be WEAK (see https://travis-ci.org/halide/Halide/jobs/118271375)